src/converters/artifact-converter.ts

```diff
@@ -0,0 +1,91 @@
+import * as aws from '@pulumi/aws';
+import * as cx from 'aws-cdk-lib/cx-api';
+import { getAccountId, getPartition, getRegion } from '@pulumi/aws-native';
+import { FileAssetManifest } from '../assembly';
+import { FileAssetPackaging } from 'aws-cdk-lib/cloud-assembly-schema';
+import { zipDirectory } from '../zip';
+import { IStackComponent } from '../types';
+
+/**
+ * ArtifactConverter
+ */
+export abstract class ArtifactConverter {
+    constructor(protected readonly host: IStackComponent) {}
```

Comment:
Why is an `IStackComponent` a `host`? What does that exactly mean? Is it the enclosing stack?

Reply:
Updated this to be a `stackComponent`.